h/static/scripts/group-forms/components/AnnotationCard.tsx

```diff
@@ -43,6 +54,24 @@ export default function AnnotationCard({ annotation }: AnnotationCardProps) {
     [config.context.group],
   );
 
+  const updateModerationStatus = useUpdateModerationStatus(annotation);
+  const [selectDisabled, setSelectDisabled] = useState(false);
```

Comment:
I recommend the state name should reflect the condition (the moderation status is being saved) rather than the action to take (disable the select). This avoids confusion if the logic to disable the select changes to be something than exactly this state in future.